oscm-rest-api-service/src/main/java/org/oscm/rest/service/CompatibleServiceResource.java

```diff
@@ -27,7 +29,7 @@
 @Stateless
 public class CompatibleServiceResource extends RestResource {
 
-  @EJB ServiceBackend sb;
+  @EJB @Setter(value = AccessLevel.PROTECTED) ServiceBackend sb;
```

Comment:
Please be consistent with formatting (either @Setter in same line or in the next line